sensor_msgs/msg/Attitude.msg

```diff
@@ -0,0 +1,14 @@
+# This message represents 3D attitude
+
+int8 ENU=1
+int8 NED=2
+int8 ENU_FLOATING=3
+int8 NED_FLOATING=4
```

Comment:
Define these right above `reference_frame_type` to make it more clear these belong to it?